queries.yaml

```diff
@@ -981,7 +981,7 @@ post_process:
       end_zoom: 9
       items_matching: { kind: locality }
       max_items: 1
-      grid_width: 3
+      grid_width: 10
```

Comment:
To 6?

Reply:
Resolved via https://github.com/tilezen/vector-datasource/pull/2040/commits/e0fe15091eec17cd2bd3cc23dfb853fd2d3d47d1.